### DISCUSSION TITLE
font error

### DISCUSSION THREAD

**Discussion by Roman-1234** [ImageMagick/ImageMagick#3247] [Help]
Hi! to all!

ImageMagick 7 win7 x64

D:\output.png -size 1600x500   -background none -fill #E9C121 -gravity center  -font bodoni caption:"EMILY BRONTË"  -geometry +2106+1950  -gravity northwest -compose over -composite D:\output.png

![cropoutput](https://user-images.githubusercontent.com/79090269/107945444-4cda0200-6f98-11eb-8826-8b1a4cb3ef1c.jpg)

What to do? Where digging?

**Comment by Roman-1234** [comment]
font supports this letter ë

**Comment by snibgo** [comment]
You don't show the complete command, which would start with "magick". Why not?

How do you run the command: at the command line, or in a BAT script, or what?

I suspect a codepage problem.

**Reply by Roman-1234** [reply]
Test via command line 
"C:\Program Files\ImageMagick\magick.exe" convert D:\output.png -gravity center -rotate 90 -size 2500x134   -background #0005 -fill #646C6E -font bodoni caption:"EMILY BRONTËò"  D:\
![output-1](https://user-images.githubusercontent.com/79090269/107960070-d4ca0700-6fac-11eb-8e1d-9682787ab007.png)
output.png

**Reply by Roman-1234** [reply]
Something wrong in Windows system?..

**Reply by Roman-1234** [reply]
This is my system erroor.
Need to reinstal win7

**Comment by snibgo** [comment]
I suspect a codepage problem. What does the "chcp" command say? Can you "chcp 65001"? Does that help?
